### PR TITLE
Update README to mention LANGUAGES option

### PR DESCRIPTION
**Pull request by StefanBruens**
partial fix for #616

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -115,18 +115,19 @@ e.g. `-DBUILD_FORMATS=html`
...
+when configuring a build with CMake, e.g. `-DLANGUAGES=it;en;de`, etc.
 
-Currently, the available languages are : `en`, `fr`, `it`, `ja`, `nl`, and `pl` however, any
+Currently, the available languages are : `ca`, `de`, `es`, `fr`, `id`, `it`,
+`ja`, `nl`, `pl`, `ru`  and `zh`, however, any
 language code can be selected. Only translated documents will be built, so for
 some languages there may only be a partial documentation output.
 
-When the **SINGLE_LANGUAGE** option is set, the package name is transformed to
+When the **LANGUAGES** option is set, the package name is transformed to
```
Comment (StefanBruens) [inline README.adoc]: Should this be dropped, or should we concatenate the languages?

Comment (nickoe) [reply README.adoc]: What do you mean by dropped?

Comment (nickoe) [reply README.adoc]: I guess we can drop this. I don't think it even works that way right now anyways.

**Comment by nickoe** [comment]
@StefanBruens are you going to do any more changes to this?

**Comment by StefanBruens** [comment]
@nickoe no further changes, should be good to go.

**Comment by nickoe** [comment]
Thank you.

**Pull request merged**